### PR TITLE
#168284777 change mentorslist page

### PR DESCRIPTION
**Pull request by fikepaci**
#### What does this PR do?
user should be able to see list of all mentors

#### Description of task completed
Given
#### When a user click on browse mentors page
Then should be able to view list of all mentors well 

#### What are the relevant pivotal tracker stories?
[#168284777](https://www.pivotaltracker.com/story/show/168284777)

### CODE REVIEW

File: ui/html/mentorslist.html [hunk 1]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-            <p class="footer-company-about">
-                <span>About free mentors</span>
-                Free Mentors is a social initiative where accomplished professionals become role
-                models to young people to provide free mentorship sessions.</p>
-
-        </div>
-    </footer>
 </body>
 
 </html>
```
Comment (oesukam) [inline ui/html/mentorslist.html:312]: Add blank line

**Review by oesukam — CHANGES_REQUESTED** [review]

File: ui/html/mentorslist.html [hunk 2]
```diff
@@ -7,29 +7,27 @@
...
     <link rel="stylesheet" href="../css/mentorunique.css">
     <link rel="stylesheet" href="../css/footer.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
 </head>
 
 <body>
     <div class="header">
         <a href="#default" class="logo">FREE MENTORS</a>
         <div class="header-right">
-            <a href="rejected.html">Rejected</a>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:16]: i deleted this link cause it was no longer give us any thing

File: ui/html/mentorslist.html [hunk 3]
```diff
@@ -7,29 +7,27 @@
...
-            <a href="accepted.html">Accepted</a>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:17]: i deleted accepted page cause also no page was no longer linked on it

File: ui/html/mentorslist.html [hunk 4]
```diff
@@ -7,29 +7,27 @@
...
-            <a href="awaiting.html">Awaiting</a>
+            <a href="./applications.html">Applicatons</a>
             <a class="active" href="mentorslist.html">Browse Mentors</a>
             <a href="../index.html">Logout</a>
         </div>
     </div>
     <div id="id02" class="modal">
             <form class="modal-content animate" action="./mentorslist.html">
                 <div class="imgcontainer">
-                    <h1>create session form</h1>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:26]: this heading was not necesarry cause i spcifie that in place holder that user can fill that form to create session

File: ui/html/mentorslist.html [hunk 5]
```diff
@@ -7,29 +7,27 @@
...
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
 </head>
 
 <body>
     <div class="header">
         <a href="#default" class="logo">FREE MENTORS</a>
         <div class="header-right">
-            <a href="rejected.html">Rejected</a>
-            <a href="accepted.html">Accepted</a>
-            <a href="awaiting.html">Awaiting</a>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:18]: i deleted also awaiting cause no page was linked on it

File: ui/html/mentorslist.html [hunk 6]
```diff
@@ -7,29 +7,27 @@
...
             <form class="modal-content animate" action="./mentorslist.html">
                 <div class="imgcontainer">
-                    <h1>create session form</h1>
+                    <h1>Create session form</h1>
                     <p>Please fill in this form to make a session request to this mentor.</p>
                     <span onclick="document.getElementById('id02').style.display='none'" class="close"
                         title="Close Modal">×</span>
                 </div>
                 <div class="container">
-                    <label for="remark"><b>questions</b></label>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:32]: i deleted this label also cause i specifiy also in placeholder of textarea that user should enter their questions there

File: ui/html/mentorslist.html [hunk 7]
```diff
@@ -38,13 +36,26 @@ <h1>create session form</h1>
         </div>
     <div class="card">
         <div class="photo">
-            <img src="../images/user2.png" class="profile-img">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:41]: i remove this image cause was no longer useful

File: ui/html/mentorslist.html [hunk 8]
```diff
@@ -94,22 +115,32 @@ <h5>by clicking <span class="black">"apply" </span> u want
 
                 Until now I've collaborated in a lot of complex projects, I've shared a lot of
                 knowledge
-                with a lot of awesome developers and I really want to help you to begin an IT career
```
Comment (fikepaci) [inline ui/html/mentorslist.html:97]: i remove them cause  there was repetion of the same words

File: ui/html/mentorslist.html [hunk 9]
```diff
@@ -94,22 +115,32 @@ <h5>by clicking <span class="black">"apply" </span> u want
...
-                or just
```
Comment (fikepaci) [inline ui/html/mentorslist.html:98]: i remove them cause  there was repetion of the same words

File: ui/html/mentorslist.html [hunk 10]
```diff
@@ -94,22 +115,32 @@ <h5>by clicking <span class="black">"apply" </span> u want
...
-                help you to improve your IT skills and get ready for the IT business.
```
Comment (fikepaci) [inline ui/html/mentorslist.html:99]: i remove them cause  there was repetion of the same words

File: ui/html/mentorslist.html [hunk 11]
```diff
@@ -94,22 +115,32 @@ <h5>by clicking <span class="black">"apply" </span> u want
...
+                with a lot of awesome developers and I really want to help you to begin an IT.
             </p></br>
             <h4><button type="submit" onclick="document.getElementById('id02').style.display='block'"
-                style="width:auto;">apply</button></h4><h4><br>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:102]: i remove them cause  there was repetion of the same links

File: ui/html/mentorslist.html [hunk 12]
```diff
@@ -94,22 +115,32 @@ <h5>by clicking <span class="black">"apply" </span> u want
...
+                style="width:auto;" && .main.display = "none">apply</button></h4><h4><br>
             <h5>by clicking <span class="black">"apply" </span> you accept and make a request that you want
                 to be a mentee of this specific mentor</h5><br>
         </div>
     </div>
     </div>
     <div class="card">
         <div class="photo">
-            <img src="../images/user2.png" class="profile-img">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:110]: i change this image with the which was un meaningful and i change it to image of a human

File: ui/html/mentorslist.html [hunk 13]
```diff
@@ -129,9 +160,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
 
                 Until now I've collaborated in a lot of complex projects, I've shared a lot of
                 knowledge
-                with a lot of awesome developers and I really want to help you to begin an IT career
```
Comment (fikepaci) [inline ui/html/mentorslist.html:132]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 14]
```diff
@@ -129,9 +160,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                or just
```
Comment (fikepaci) [inline ui/html/mentorslist.html:133]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 15]
```diff
@@ -129,9 +160,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                help you to improve your IT skills and get ready for the IT business.
```
Comment (fikepaci) [inline ui/html/mentorslist.html:134]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 16]
```diff
@@ -142,9 +171,21 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
     </div>
     <div class="card">
         <div class="photo">
-            <img src="../images/user2.png" class="profile-img">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:145]: i change this image to image of human

File: ui/html/mentorslist.html [hunk 17]
```diff
@@ -164,9 +205,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
 
                 Until now I've collaborated in a lot of complex projects, I've shared a lot of
                 knowledge
-                with a lot of awesome developers and I really want to help you to begin an IT career
```
Comment (fikepaci) [inline ui/html/mentorslist.html:167]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 18]
```diff
@@ -164,9 +205,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                or just
```
Comment (fikepaci) [inline ui/html/mentorslist.html:168]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 19]
```diff
@@ -164,9 +205,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                help you to improve your IT skills and get ready for the IT business.
```
Comment (fikepaci) [inline ui/html/mentorslist.html:169]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 20]
```diff
@@ -178,11 +217,23 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
     </div>
     <div class="card">
         <div class="photo">
-            <img src="../images/user2.png" class="profile-img">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:181]: i change this image with image of real person

File: ui/html/mentorslist.html [hunk 21]
```diff
@@ -178,11 +217,23 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
+                         <input type="radio" name="rating" value="5" ><span class="star"></span>
+                             <input type="radio" name="rating" value="4" checked><span class="star"></span>
+                             <input type="radio" name="rating" value="3"><span class="star"></span>
+                             <input type="radio" name="rating" value="2"><span class="star"></span>
+                             <input type="radio" name="rating" value="1"><span class="star"></span>
+                         </span>
+                         </form>
+                     </span>
             <a href="mentorunique.html">
-                <p class="name">mc tino</p>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:185]: change first letters to upper case

File: ui/html/mentorslist.html [hunk 22]
```diff
@@ -200,9 +251,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
 
                 Until now I've collaborated in a lot of complex projects, I've shared a lot of
                 knowledge
-                with a lot of awesome developers and I really want to help you to begin an IT career
```
Comment (fikepaci) [inline ui/html/mentorslist.html:203]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 23]
```diff
@@ -200,9 +251,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                or just
```
Comment (fikepaci) [inline ui/html/mentorslist.html:204]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 24]
```diff
@@ -200,9 +251,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                help you to improve your IT skills and get ready for the IT business.
```
Comment (fikepaci) [inline ui/html/mentorslist.html:205]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 25]
```diff
@@ -214,9 +263,21 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
     </div>
     <div class="card">
         <div class="photo">
-            <img src="../images/user2.png" class="profile-img">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:217]: i change this with image of real person

File: ui/html/mentorslist.html [hunk 26]
```diff
@@ -236,9 +297,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
 
                 Until now I've collaborated in a lot of complex projects, I've shared a lot of
                 knowledge
-                with a lot of awesome developers and I really want to help you to begin an IT career
```
Comment (fikepaci) [inline ui/html/mentorslist.html:239]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 27]
```diff
@@ -236,9 +297,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                or just
```
Comment (fikepaci) [inline ui/html/mentorslist.html:240]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 28]
```diff
@@ -236,9 +297,7 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                help you to improve your IT skills and get ready for the IT business.
```
Comment (fikepaci) [inline ui/html/mentorslist.html:241]: i changed this word with short meaning ful words

File: ui/html/mentorslist.html [hunk 29]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
         </div>
     </div>
     </div>
-    <footer class="footer-distributed">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:251]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 30]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-
```
Comment (fikepaci) [inline ui/html/mentorslist.html:252]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 31]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-        <div class="footer-left">
-            <h3>FREE<span> MENTORS</span></h3>
-
-            <p class="footer-links">
-                <a href="#">Home</a>
-                |
```
Comment (fikepaci) [inline ui/html/mentorslist.html:258]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 32]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <a href="#">Blog</a>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:259]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 33]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                |
```
Comment (fikepaci) [inline ui/html/mentorslist.html:260]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 34]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <a href="#">About</a>
-                |
-                <a href="#">Contact</a>
-            </p>
-
-
```
Comment (fikepaci) [inline ui/html/mentorslist.html:266]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 35]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-        </div>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:267]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 36]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-
```
Comment (fikepaci) [inline ui/html/mentorslist.html:268]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 37]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-        <div class="footer-center">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:269]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 38]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-            <div>
-
-                <p><span>location 102 - kigali,
-                        reg. No. A - 1, kicukiro - 1</span>
-                </p>
-            </div>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:275]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 39]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-
```
Comment (fikepaci) [inline ui/html/mentorslist.html:276]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 40]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-            <div>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:277]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 41]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <p>phone number<p>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:278]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 42]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                        <p>+250780607091</p>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:279]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 43]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-            </div>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:280]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 44]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-            <div>
-                <p><a href="freementors@yahoo.com">freementors@yahoo.com</a></p>
-                <p class="footer-company-name">&copy free mentors students</p>
-            </div>
-        </div>
-        <div class="footer-right">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:286]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 45]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-            <p class="footer-company-about">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:287]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 46]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <p><span>location 102 - kigali,
-                        reg. No. A - 1, kicukiro - 1</span>
-                </p>
-            </div>
-
-            <div>
-                <p>phone number<p>
-                        <p>+250780607091</p>
-            </div>
-            <div>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:281]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 47]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <p><a href="freementors@yahoo.com">freementors@yahoo.com</a></p>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:282]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 48]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <p class="footer-company-name">&copy free mentors students</p>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:283]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 49]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-            </div>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:284]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 50]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-        </div>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:285]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 51]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-        <div class="footer-right">
-            <p class="footer-company-about">
-                <span>About free mentors</span>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:288]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 52]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                Free Mentors is a social initiative where accomplished professionals become role
```
Comment (fikepaci) [inline ui/html/mentorslist.html:289]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 53]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                models to young people to provide free mentorship sessions.</p>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:290]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 54]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-
```
Comment (fikepaci) [inline ui/html/mentorslist.html:291]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 55]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-        </div>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:292]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 56]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-    </footer>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:293]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 57]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <a href="#">About</a>
-                |
-                <a href="#">Contact</a>
-            </p>
-
-
-        </div>
-
-        <div class="footer-center">
-            <div>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:270]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 58]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-
```
Comment (fikepaci) [inline ui/html/mentorslist.html:271]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 59]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <p><span>location 102 - kigali,
```
Comment (fikepaci) [inline ui/html/mentorslist.html:272]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 60]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                        reg. No. A - 1, kicukiro - 1</span>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:273]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 61]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                </p>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:274]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html (hunk 38 above)
Comment (fikepaci) [inline ui/html/mentorslist.html:275]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 62]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-
-        <div class="footer-left">
-            <h3>FREE<span> MENTORS</span></h3>
-
-            <p class="footer-links">
-                <a href="#">Home</a>
-                |
-                <a href="#">Blog</a>
-                |
-                <a href="#">About</a>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:261]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 63]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                |
-                <a href="#">Contact</a>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:263]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 64]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-        <div class="footer-left">
-            <h3>FREE<span> MENTORS</span></h3>
-
-            <p class="footer-links">
-                <a href="#">Home</a>
-                |
-                <a href="#">Blog</a>
-                |
-                <a href="#">About</a>
-                |
```
Comment (fikepaci) [inline ui/html/mentorslist.html:262]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 65]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <a href="#">Contact</a>
-            </p>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:264]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 66]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-
```
Comment (fikepaci) [inline ui/html/mentorslist.html:265]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html (hunk 35 above)
Comment (fikepaci) [inline ui/html/mentorslist.html:267]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 67]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
         </div>
     </div>
     </div>
-    <footer class="footer-distributed">
-
-        <div class="footer-left">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:253]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 68]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-            <h3>FREE<span> MENTORS</span></h3>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:254]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 69]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-
```
Comment (fikepaci) [inline ui/html/mentorslist.html:255]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 70]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-            <p class="footer-links">
```
Comment (fikepaci) [inline ui/html/mentorslist.html:256]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html [hunk 71]
```diff
@@ -248,49 +307,6 @@ <h5>by clicking <span class="black">"apply" </span> you accept and make a reques
...
-                <a href="#">Home</a>
```
Comment (fikepaci) [inline ui/html/mentorslist.html:257]: i saw it un necessary and i remove it

File: ui/html/mentorslist.html (hunk 31 above)
Comment (fikepaci) [inline ui/html/mentorslist.html:258]: i saw it un necessary and i remove it

**Review by fikepaci — COMMENTED** [review]
i change images with images of real person , i add stars rate for each mentor, i remove footer cause i see not necessary for this page , some text i changed them to short meaning words

**Comment by fikepaci** [comment]
i change un meaningful images with image of real persons for each mentor in a list, i added stars rate for each mentor, theri is some text i changed into meaningful and short words, i remove footer cause i see it not necessary

**Review by oesukam — APPROVED** [review]

**Pull request merged**